django_freeradius/utils.py

```diff
@@ -0,0 +1,12 @@
+import random
+import string
+
+from django_freeradius.settings import DEFAULT_PASSWORD_LENGTH
+
+
+def generate_password(length=None, chars=None):
```

Comment:
we probably don't need this at all, check this out:

```python
>>> from django.utils.crypto import get_random_string
>>> get_random_string(length=8)
'2sEkWhzp'
```

Reply:
Ah. Looks good